### PR TITLE
fix(core): module resolution in webpack 5 broken due this.context poi…

### PR DESCRIPTION
**Pull request by Delagen**
module resolution in webpack 5 broken due this.context points to module entry directory

This PR contains a:

- [x] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case

Broken module resolution in webpack 5

### Breaking Changes

Seems missing )

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ module.exports.pitch = function pitch(remainingRequest) {
   // builds in different directories.
   const newRequestPath = remainingRequest.replace(
     this.resourcePath,
-    `.${path.sep}${path.relative(this.context, this.resourcePath)}`
+    `.${path.sep}${path.relative(this.rootContext, this.resourcePath)}`
```
Comment (evilebottnawi) [inline lib/index.js]: Can we use `this.rootContext || this.context` to avoid breaking change?

Comment (Delagen) [reply lib/index.js]: I think we can )

Comment (Delagen) [reply lib/index.js]: Can you view also https://github.com/webpack-contrib/val-loader/pull/37 ?

Comment (evilebottnawi) [reply lib/index.js]: @Delagen in my todo, release will be in this week

Comment (Delagen) [reply lib/index.js]: Moved to https://github.com/webpack-contrib/expose-loader/pull/92

**Pull request closed without merging**